src/web/modules/services/detail.njk

```diff
@@ -114,6 +116,14 @@
       href: "/stripe/basic/create?service=" + service.external_id
       })
     }}
+    <h2 class="govuk-heading-s">Test account</h2>
+    <p class="govuk-body">Create Stripe test account for service:</p>
+    {{ govukButton({
+      text: "Setup test Stripe account",
+      classes: "govuk-button--secondary",
+      href: "/stripe/basic/create-test-account?service=" + service.external_id
+      })
+    }}
 
     <p>
       <a class="govuk-link" href="/stripe/create?service={{ service.external_id }}">
```

Comment:
It's not part of this functionality change but it looks like the code for this link has been removed but the link still exists. It should be removed.

Reply:
Didn't notice, although added button top of this line. Removed